src/module/addRemove.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-loop-func */
+let todoList = [];
+let isEditing = false;
+let todoEdit = null;
+
+const saveData = () => {
+  localStorage.setItem('todoslist', JSON.stringify(todoList));
+};
+
+const getStorageData = () => {
+  const localFormData = JSON.parse(localStorage.getItem('todoslist'));
+  if (localFormData == null) {
```

Comment:
1. Please use "===" instead of "==" validation
2. Please validate `null `and ` undefined ` object as describes in [Common mistakes](https://www.w3schools.com/js/js_mistakes.asp)